SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/GridBasedLBStrategy/GridBasedLBStrategyTest.cpp

```diff
@@ -348,3 +348,12 @@ GRIDBASEDLBSTRATEGY_TEST(GIVEN_two_cells_WHEN_actor_in_one_cell_THEN_strategy_re
 
 	return true;
 }
+
+GRIDBASEDLBSTRATEGY_TEST(zz_CleanUp_GridBasedLBStrategyTest)
+{
+	// Test are ran in alphabetical order and we don't want to make the tests slow by stopping the deployment after each
```

Comment:
```suggestion
	// Tests are run in alphabetical order and we don't want to make the tests slow by stopping the deployment after each
```
or is it "ran"?

Reply:
that's my english AS level down the drain 😅